app/services/cognito/sign_in_user.rb

```diff
@@ -51,8 +51,7 @@ def success?
     def initiate_auth
       cognito_common = Cognito::Common.new
       client_creds = cognito_common.get_client_credentials(client_id)
-
-      if client_creds.user_pool_client.explicit_auth_flows.include? 'USER_PASSWORD_AUTH'
+      if (client_creds.user_pool_client.explicit_auth_flows & %w[USER_PASSWORD_AUTH ALLOW_USER_PASSWORD_AUTH]).any?
```

Comment:
Why did you need to do this?

Reply:
it looks like the new app clients do not return USER_PASSWORD_AUTH explicit auth flow anymore, but they return ALLOW_USER_PASSWORD_AUTH instead for the 'Enable username-password (non-SRP) flow'